examples/hybrid/plot_scaling_results.jl

```diff
@@ -77,22 +77,31 @@ ENV["GKSwstype"] = "100"
 Plots.GRBackend()
 plt1 = plot(
     log2.(nprocs_clima_atmos),
-    sypd_clima_atmos,
+    log.(sypd_clima_atmos),
```

Comment:
Why we don't want to use the same bases?
I wouldn't mix different logarithmic bases on the different axes. It is common to either use a semilog (or lin-log) plot as you had before or a log-log plot, but not a log-log2 plot where one axis has base 2 and the other one natural base.